### DISCUSSION TITLE
Github desktop installer broken

### DISCUSSION THREAD

**Discussion by IggyDinosaur** [community/community#146948] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Tried to update my github desktop that was working with the newest version and it's completely busted.  I do not see any error messages.  I did uninstall the program + remove the local app directory files associated with it.  No idea how to fix it at this point as the installer itself seems busted.  It does drop a bunch of the files in the local app directory, but it does not run.

Suggestions on where to look or how to fix?

**Comment by steveward** [comment]
@IggyDinosaur can you open an issue in https://github.com/desktop/desktop/issues so that we can assist you further there? Thanks!
